RallyHereIntegration/RallyHereIntegration.uplugin

```diff
@@ -1,8 +1,8 @@
 {
     "FileVersion": 3,
     "FriendlyName": "RallyHere Integration",
-    "Version": 1,
-    "VersionName": "1.0",
+    "Version": 2,
+    "VersionName": "11.0",
```

Comment:
Should this be the full semver version number?

Reply:
Probably - I believe the VersionName is just a string, the version number above is the one used for comparison.